journal-letters/modernized/03-chapter-1.adoc

```diff
@@ -156,7 +157,10 @@ and in conclusion I was favored in fervent supplication.
 
 At Gilmanton he wrote to his wife and children as follows:--
```

Comment:
I'm not sure why these are not in brackets, but since they are narrator comments, I think they need to be [.offset] with brackets around them. I saw a bunch of these little narrator comments introducing letters or excerpts from things throughout these first two chapters.  Let's offset them, put them in brackets, and kill the long dash after the colon at the end.